app/controllers/reports/regions_controller.rb

```diff
@@ -37,10 +37,21 @@ def show
     @with_ltfu = with_ltfu?
 
     @children = @region.reportable_children
+
+    repository = Reports::Repository.new(@children, periods: @period, with_exclusions: report_with_exclusions?)
```

Comment:
The repository lets us grab the data for _just_ the period we are interested in for all the children regions.